specs/commandline/sign.md

```diff
@@ -31,9 +31,11 @@ Flags:
   -d,  --debug                      debug mode
   -e,  --expiry duration            optional expiry that provides a "best by use" time for the artifact. The duration is specified in minutes(m) and/or hours(h). For example: 12h, 30m, 3h20m
   -h,  --help                       help for sign
+       --id string                  key id (required if --plugin is set)
```

Comment:
Is `keyid` a better name for it?

Reply:
Don't have strong opinion here but was trying to keep it consistent with `keys add` command